src/Serokell/Util/Verify.hs

```diff
@@ -78,19 +80,18 @@ verifyGeneric errors
 -- | Format VerificationRes in a pretty way using all errors messages
 -- for VerFailure.
 buildVerResFull :: VerificationRes -> B.Builder
-buildVerResFull VerSuccess          = buildVerResImpl Nothing
-buildVerResFull (VerFailure errors) = buildVerResImpl $ Just errors
+buildVerResFull VerSuccess          = "success"
+buildVerResFull (VerFailure errors) = buildVerResImpl errors
 
 -- | Format VerificationRes in a pretty way using only first message
 -- for VerFailure.
 buildVerResSingle :: VerificationRes -> B.Builder
-buildVerResSingle VerSuccess          = buildVerResImpl Nothing
+buildVerResSingle VerSuccess          = "success"
 -- [SU-1] Use NonEmpty instead of unsafe head.
-buildVerResSingle (VerFailure errors) = buildVerResImpl $ Just $ [head errors]
+buildVerResSingle (VerFailure errors) = buildVerResImpl $ head errors :| []
 
-buildVerResImpl :: Maybe [T.Text] -> B.Builder
-buildVerResImpl Nothing       = "success"
-buildVerResImpl (Just errors) =
+buildVerResImpl :: NonEmpty Text -> B.Builder
+buildVerResImpl errors =
     "failure: " `mappend` listBuilder @Text @Text @Text "[" "; " "]" errors
 
 -- | Formatter based on buildVerResFull.
```

Comment:
Could you please remove `verResFullF` and `verResSingleF` formatters which use `formatting` library? We're planning to move to `fmt` library so we can begin to remove `formatting` from modules we're working on. Well, actually, it's better to not just remove these two formatters but also rename `buildVerResFull` and `buildVerResSingle` to  `verResFullF` and `verResSingleF`. And you also need to remove `Formatting` import from this module as a result of this refactoring.